static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -6,36 +8,38 @@
 }
 
 .new-header__logo {
+    padding-top: ($gs-baseline / 2);
+    padding-bottom: ($gs-baseline / 2);
 
     svg {
         // Removes extra spacing
         display: block;
+        padding-right: ($gs-gutter / 2);
 
         // Aspect ratio: 16:3
-        @include mq($until: mobileLandscape) {
-            width: 50vw;
+        @include mq($until: $mobileMedium) {
+            width: 170px;
             // height: auto doesn't work in Safari
-            height: calc(3/16 * 46vw);
-            padding: ($gs-baseline / 2) 0;
-            min-width: 147px;
-            min-height: 28px;
+            height: calc(3/16 * 170px);
         }
 
-        @include mq(mobileLandscape) {
-            width: 240px;
-            // height: auto doesn't work in Safari
-            height: calc(3/16 * 240px);
-            padding: ($gs-baseline / 2) ($gs-gutter / 2);
+        @include mq($from: $mobileMedium, $until: mobileLandscape) {
+            width: 225px;
+            height: calc(3/16 * 225px);
```

Comment:
@sndrs Thanks for taking a look! I've added height auto now as a fallback for browsers like opera mini etc that don't support calc. The logo dimensions don't correspond to grid system, the numbers are purely for the visual weight of the header.